wings/1.0/configuration.md

```diff
@@ -0,0 +1,31 @@
+# Additional Configuration
+
+[[toc]]
+
+::: warning
+These are advanced configurations for the daemon. You risk breaking your daemon and making containers un-usable if
```

Comment:
I tend to refer to "the daemon" as "Wings" (proper noun) for the sake of avoiding confusion with 0.6 and for consistency.

Reply:
Fixed ^^